README.md

```diff
@@ -36,15 +36,27 @@ Der Server bietet sowohl für die App als auch für den Bot die notwendigen API
 Der Source-Code und Dokumentation des Servers ist in diesem Repo zu finden.
 
 #### Install and Start
-Voraussetzung ist ein installiertes python 3.8 package
-- `sudo ./manage.sh install` um den server zu installieren
-  (sudo only necessary to install graphviz to display story graph)
-- `./manage.sh start` um den server zu starten
-- für weitere befehle `./manage.sh help` benutzen
+Voraussetzung ist ein Linux System mit installiertem python 3.8
+1. `./manage.sh install` um den server zu installieren
+1. `./manage.sh start` um den server zu starten
+1. für weitere befehle `./manage.sh help` benutzen
+1. siehe Bot repo für Installation und Start des Bots
+1. nach Start von Server und Bot kann die App und damit das Spiel gestartet werden
 
 #### Architektur-Überblick
 Der Server wird durch eine Flask App (`/app`) implementiert. Diese verwaltet verschiedene API-Endpunkte (`/app/routes`). Außerdem wird eine Datenbank verwaltet. Deren ORM wird in `/app/models` implementiert. Der Story-Inhalt und der Story-verwaltende Code liegt unter `/app/story`.  
 
+#### Nutzung ohne App
+Unsere App ist sehr datenhungrig. Um das Spiel zu testen, ohne die App zu installieren, haben wir ein Postman package erstellt, welches die App immitiert mit dem sich das Spiel durchspielen lässt (die Spielerfahrung leidet darunter jedoch deutlich). Die Nutzung ist wie folgt:
+1. `detective-game-no-app-walkthrough.json` mit Postman importieren
```

Comment:
Request `arrest_suspect` and `observe_suspect` is not completed by sending the mock data. Some error here?
The entry is made to the db, but the finish request returns `false`.

The code looks like `$timestamp` is sent, but a time between `15:55` and `16:05` is needed

Reply:
that's correct... I will make concrete timestamps for all requests

Reply:
Could you explain the problem a bit more detailed? Did not understand completely what the problem is

Reply:
Suppose we have location Data from the user at the hbf from 3 weeks ago, we don't want that data to complete the task. this is why the server only takes data into account that is not older than e.g. 5min, which would mean the user has to be at hbf NOW.
I guess we could just timestamps that lie in the future though...

Reply:
The way `wait` is implemented is a bit lazy - it works if data is sent on creation, but otherwise not. I will do a refactor on another branch

Reply:
done. please merge #85, check again, if the postman package works and then merge this MR